core/src/main/scala/kafka/utils/TopicFilter.scala

```diff
@@ -57,7 +54,6 @@ case class Whitelist(rawRegex: String) extends TopicFilter(rawRegex) {
   }
 }
 
-@deprecated("This class has been deprecated and will be removed in a future release.", "0.11.0.0")
 case class Blacklist(rawRegex: String) extends TopicFilter(rawRegex) {
```

Comment:
It seems that `Blacklist` is only used in test. Do we still need this?

Reply:
Good point, we only need to keep `Whitelist`.